routes/localisation.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+
+const router = express.Router();
+const { db } = require('../db-config');
+
+router.get('/', async (req, res) => {
+  try {
+    const [location] = await db.query(`
+    SELECT id, type, name, streetNumber, street, postalCode, city, latitude, longitude, phone
+    FROM location
+  `);
+    console.log(location);
```

Comment:
I think I saw this in an other PR. plz don't push the same work into different PRs !